frontend/src/pages/search/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
`1:8  Error: 'React' is defined but never used.  @typescript-eslint/no-unused-vars`
```suggestion
import { useState, useEffect } from 'react';
```